build.gradle

```diff
@@ -0,0 +1,100 @@
+import java.util.regex.Pattern
+
+plugins {
+    id 'java-library'
+}
+
+ant.importBuild('build.xml') { String oldTargetName ->
```

Comment:
With the introducion of gradle id like to get rid of the old. 
I don't see any good reason for holding onto the ant build-system. 
Could you please remove them?

Reply:
Removed now

Reply:
sweet!